### DISCUSSION TITLE
Not scroll to top of the page when use window.scrollTo(0 ,0 ) on mounted()

### DISCUSSION THREAD

**Discussion by RupeshSeth** [quasarframework/quasar#9941] [General - Components / Directives / etc]
Hi all,

I am using 5 cards in 5 component and all components are inside one layout.
some cards are more height and some are small in height. when we go from 1st card to 2nd card the without scroll the page. the behaviour is fine. but if i scroll down the 1st card and then click next on first card the position of the scroll is still the same and the card appear from in between of the screen not from top. I used window.scrollTo(0, 0) on mounted of the component as well as scrollBehaviour but noting happened. i tried more solution from the internet but I am failed to solved. Can any one please let me know how the body scroll set to 0 0 when component load.

**Comment by pdanpdan** [comment]
Add a codepen
